rasa/nlu/data_router.py

```diff
@@ -361,33 +362,62 @@ def training_errback(failure):
     def evaluate(self,
                  data: Text,
                  project: Optional[Text] = None,
-                 model: Optional[Text] = None) -> Dict[Text, Any]:
+                 model: Optional[Text] = None) -> Deferred:
         """Perform a model evaluation."""
 
+        logger.debug("Evaluation request received")
+
+        if self._training_processes <= self._current_training_processes:
```

Comment:
I'm not sure we should be checking this given that it's not a training process

Reply:
I have now renamed everything to `pool_process` terminology.  With that in, you didn't mean to skip this check entirely did you?

Reply:
no the check should still be there, I just meant using different terminology